### DISCUSSION TITLE
how can i get a galaxy brain achievement

### DISCUSSION THREAD

**Discussion by mbezerraa** [Schweinepriester/github-profile-achievements#120] [Q&A]

**Comment by mrcsxsiq** [comment]
You must have two accepted answers in discussions.

**Comment by Keyrxng** [comment]
It looks like you may have just earned that achievement within the last couple of hours so gz.

**Comment by thenithinbalaji** [comment]
You must have 2 accepted answers in discussions. 
For example, if you mark this answer as accepted, I will get a point.
